src/main/java/group1/mips_simulator/SimApplication.java

```diff
@@ -13,7 +13,7 @@ public void start(Stage stage) throws IOException {
         FXMLLoader fxmlLoader = new FXMLLoader(SimApplication.class.getResource("hello-view.fxml"));
         Scene scene = new Scene(fxmlLoader.load(), 320, 240);
         stage.setTitle("Hello!");
-        stage.setScene(scene);
+        stage.setsScene(scene);
```

Comment:
Not sure what this change was, I'll revert it. Probably a `ctrl+s` but missing the `ctrl` part. Oops